uniswap-v2/proto/uniswap_v2.proto

```diff
@@ -2,37 +2,78 @@ syntax = "proto3";
 
 package messari.uniswap_v2;
 
-message ERC20Tokens {
-  repeated ERC20Token items = 1;
-}
-
-message ERC20Token {
-  string address = 1;
-  string name = 2;
-  string symbol = 3;
-  uint64 decimals = 4;
-}
+// https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Factory.sol
+// https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Pair.sol
 
-message Pairs {
-  repeated Pair items = 1;
-}
+// event PairCreated(address indexed token0, address indexed token1, address pair, uint);
+message PairCreatedEvent {
+  string tx_hash = 1;
+  uint32 log_index = 2;
 
-message Pair {
-  string name = 1;
-  string address = 2;
   string token0 = 3;
   string token1 = 4;
+  string pair = 5;
 }
 
 message PairCreatedEvents {
   repeated PairCreatedEvent items = 1;
 }
 
-message PairCreatedEvent {
-  uint64 block_number = 1;
-  uint64 block_timestamp = 2;
+// event Mint(address indexed sender, uint amount0, uint amount1);
+message MintEvent {
+  string tx_hash = 1;
+  uint32 log_index = 2;
 
-  string token0 = 3;
-  string token1 = 4;
-  string pair = 5;
+  string sender = 3;
+  uint64 amount0 = 4;
```

Comment:
an amount could go beyond MAX_UINT64 easily, shall we do `string` instead?

Same applies to all amount fields